frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -157,8 +157,15 @@ const renderChromiumUsage: CellRenderer = (
   _options,
 ) => {
   if (feature.usage?.chromium?.daily && feature.usage.chromium.daily > 0) {
-    // Format to display percentage with single decimal e.g. 0.8371 -> 83.7%.
-    return html`${(feature.usage.chromium.daily * 100).toFixed(1)}%`;
+    // If the feature has some usage, but the usage is less than 0.1%,
+    // display it that way.
+    if (feature.usage.chromium.daily < 0.01) {
```

Comment:
I do like this logic for very small numbers. Can you adjust it and the comment according to my comment above.